### PR TITLE
Upgrade thor to 0.20.0

### PR DESCRIPTION
**Pull request by rafaelfranca**
Thor 0.20.0 was released so I'm upgrading bundler to the newest version.

### CODE REVIEW

File: lib/bundler/vendor/thor/lib/thor/util.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def find_by_namespace(namespace)
       end
 
       # Receives a constant and converts it to a Bundler::Thor namespace. Since Bundler::Thor
-      # commands can be added to a sandbox, this method is also responsible for
```
Comment (segiddins) [inline lib/bundler/vendor/thor/lib/thor/util.rb]: this is a new typo

**Comment by segiddins** [comment]
Yay!

**Comment by segiddins** [comment]
```
An error occurred while loading ./spec/bundler/cli_spec.rb.
Failure/Error: all_commands["help"].disable_class_options = false
NoMethodError:
  undefined method `disable_class_options=' for #<Bundler::Thor::Command:0x00000001cccee0>
# ./spec/bundler/cli_spec.rb:2:in `<top (required)>'
```

**Comment by segiddins** [comment]
https://travis-ci.org/bundler/bundler/jobs/251510586#L672

**Comment by rafaelfranca** [comment]
I'm still not sure how to proceed with that failure. I'd prefer to not require any changes on bundler but the change on thor implements the feature that bundler was already implementing, an sane behavior when we have a conflict generating the files. The difference is that new thor behavior doesn't stop the execution of the command in the first conflict, while the bundler behavior does.

I'm fine with reverting thor's change, but maybe the new thor behavior is better than the current bundler behavior?

**Comment by segiddins** [comment]
@indirect thoughts?

**Comment by indirect** [comment]
Is the new behavior in thor to print `file_conflict` and silently continue? That seems okay to me.

**Comment by rafaelfranca** [comment]
> Is the new behavior in thor to print file_conflict and silently continue? That seems okay to me.

Yes. It will show a `file_conflict` but will continue to the next command.

**Comment by segiddins** [comment]
that one seems fine with an updated test then, but there is another test that's now raising `ENOTDIR`

**Comment by segiddins** [comment]
Updated specs, should pass now

**Comment by segiddins** [comment]
@rafaelfranca all green 🎆

**Comment by segiddins** [comment]
@rafaelfranca anything else you need from us?

**Comment by rafaelfranca** [comment]
Sorry. I'll release it tomorrow

**Comment by rafaelfranca** [comment]
I just released 0.20.0 and updated this PR. It is ready to go

**Comment by segiddins** [comment]
Yay! @bundlerbot r+

**Pull request merged**